lib/components/Balloon/Balloon.module.scss

```diff
@@ -6,9 +6,9 @@ $balloon-min-width: 48*$grid-size;
 
 .balloon-dropdown {
     display: inline-flex;
-    box-shadow: var(--shadow-balloon);
-    background-color: var(--color-bg-balloon);
-    border: 1px solid var(--color-border-controls);
+    box-shadow: 0 8px 10px 0 var(--color-dark-transparent-tenth);

```

Comment:
Use multiples of grid size?

Reply:
this where the values that were there before in `var(--shadow-balloon)` I just moved them here to standardize var uses to just colors and not the whole box-shadow.

In theory there is a set of standard box-shadows (depth variables), I can take a look at using those here.